### PR TITLE
chore: TypeScript <Label />

### PR DESCRIPTION
**Pull request by mistercrunch**
Tried @rusackas 's approach of bringing a react-bootstrap component in Storybook just for fun.

One thought is around the fact that labels are used in controls, and that it calls for eventually having to refactor components into its own package that other packages can depend on. 

Screen Shot 2020-08-05 at 5 42 40 PM
Screen Shot 2020-08-05 at 5 42 30 PM

### CODE REVIEW

**Comment by rusackas** [comment]
I believe I've fixed last test... if this gets merged in, remind me to open another PR and use the new `syledMount` helper all over the place.

**Comment by rusackas** [comment]
Ugh... this python pre-commit hook is failing on a random text file. I'm fixing it in this in https://github.com/apache/incubator-superset/pull/10515

**Comment by rusackas** [comment]
🏷 storybook

**Review by rusackas — APPROVED** [review]
Up to you if you wanna get the Emotion stuff in, or leave it for a subsequent PR, but this thing seems to be workin' for now.

**Comment by nytai** [comment]
I was just working with labels and wanted to make some tweaks. Having this would make it easier to add other styles to the label.

File: superset-frontend/src/components/Label/label.stories.jsx [hunk 1]
```diff
@@ -30,16 +30,16 @@ export default {
 const bsStyleKnob = {
   label: 'Types',
   options: {
-    Danger: 'danger',
-    Warning: 'warning',
-    Success: 'success',
-    Default: 'default',
+    danger: 'danger',
+    warning: 'warning',
+    cuccess: 'success',
```
Comment (rusackas) [inline superset-frontend/src/components/Label/label.stories.jsx:35]:
```suggestion
    success: 'success',
```

Comment (mistercrunch) [reply superset-frontend/src/components/Label/label.stories.jsx:35]: well that wasn't a success

**Pull request merged**